harness/determined/deploy/healthcheck.py

```diff
@@ -53,3 +53,37 @@ def wait_for_master_url(
     finally:
         if polling:
             print()
+
+
+def wait_for_genai_url(
+    master_url: str,
+    timeout: int = DEFAULT_TIMEOUT,
+    cert: Optional[certs.Cert] = None,
+) -> None:
+    POLL_INTERVAL = 2
+    polling = False
```

Comment:
I am not sure about the purpose of this flag.

Would we not want to poll until timeout or first success, whichever comes first?

Reply:
I borrowed this from the [master health check](https://github.com/determined-ai/determined/blob/8cd8b210f36366fb917ca6870dda2eea504755e8/harness/determined/deploy/healthcheck.py#L29-L55) without rewriting the two. 

the goal of the polling flag is purely for rendering purposes to make sure we clean up and close the output line with a newline and final msg.